src/main/java/core/basesyntax/bookstore/repository/book/BookRepository.java

```diff
@@ -1,9 +1,20 @@
 package core.basesyntax.bookstore.repository.book;
 
 import core.basesyntax.bookstore.model.Book;
+import core.basesyntax.bookstore.model.Category;
+import java.util.Optional;
+import java.util.Set;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 
 public interface BookRepository extends JpaRepository<Book, Long>,
         JpaSpecificationExecutor<Book> {
+    @Override
```

Comment:
```suggestion
```

Reply:
![image](https://github.com/user-attachments/assets/34c5647c-d917-457d-b2ca-c0ee98f03724)
but I do override this method since it declared in CrudRepository first